packages/demo/src/components/examples/SingleSelectExamples.tsx

```diff
@@ -152,6 +162,16 @@ const SingleSelectConnectedExamples: React.ComponentType = () => (
         <Section level={3} title="A single select with an error message to show with a button">
             <SingleSelectWithMessageExample />
         </Section>
+        <Section level={3} title="A single select with non-empty validation">
+            <SingleSelectWithNonEmptyValue id="single-select-non-empty" items={defaultItems} canClear />
+            <div>
+                <ValidationMessage id="single-select-non-empty" />
+            </div>
+        </Section>
+        <Section level={3} title="A single select with dirty management">
+            <SingleSelectWithDirty id="select-dirty" items={defaultItems} canClear />
+            <SaveButton enabled validationIds={['select-dirty']} name="An example button bound to the select" />

```

Comment:
layout is a bit whack:
![image](https://user-images.githubusercontent.com/33127602/99437006-d6b5f880-28df-11eb-926d-52a0b5f1b69c.png)

Reply:
Should be fixed with: https://github.com/coveo/react-vapor/pull/1729